packages/entities/entities-shared/src/components/entity-form-section/EntityFormSection.vue

```diff
@@ -1,7 +1,9 @@
 <template>
-  <fieldset
+  <div
+    :aria-labelledby="legendID"
```

Comment:
```suggestion
    :aria-labelledby="title ? legendID : undefined"
```

issue: if title is not provided, it's not gonna be rendered